GetCovid19Statistics/README.md

```diff
@@ -0,0 +1,16 @@
+# Retrieving Covid-19 Statistics country-wise
+
```

Comment:
Run on Dcoder button should not be removed, regenerate the Readme with dcoder-cli